supplementary/alica_ros1/alica_tests/include/alica_tests/TestTracing.h

```diff
@@ -3,8 +3,12 @@
 #include <cassert>
 #include <engine/IAlicaTrace.h>
 #include <optional>
+#include <sstream>
 #include <string>
```

Comment:
+<string_view>?

Reply:
already included in IAlicaTrace.h

Reply:
Yeah, in other case it won't compile =)
But in general better include, just to be independent

Reply:
I guess here it is ok not to because string_view is part of the pure virtual function API which this class overrides, so its safe to assume string_view will be included.